microk8s-resources/wrappers/addon-lists.yaml

```diff
@@ -198,3 +198,11 @@ microk8s-addons:
       check_status: "pod/keda-operator"
       supported_architectures:
       - amd64
+        
+    - name: "falco"
+      description: "enable falco - for runtime security and audit logs"
+      version: "0.26.2"
+      check_status: "pod/falco"
+      supported_architectures:
+      - arm64
```

Comment:
Is deployments on ARM64 tested enough? Looking at [1, 2,3] it is not clear to me what the state of ARM64 builds/configurations is.

Maybe we should have this addon for amd64 for now?

[1] https://github.com/falcosecurity/falco/issues/520
[2] https://github.com/falcosecurity/falco/pull/1442
[3] https://github.com/xUnholy/k8s-gitops/pull/152